drivers/core_demographics_report/app/models/core_demographics_report/details.rb

```diff
@@ -64,6 +70,12 @@ def columns_for(key)
       detail[:columns]
     end
 
+    def detail_columns_for_export(key)
+      return header_for(key) if GrdaWarehouse::Config.get(:include_pii_in_detail_downloads)
+
+      columns_for(key) - [c_t[:FirstName], c_t[:LastName], c_t[:DOB]]
```

Comment:
I'm concerned about the fragility of this...
Maybe make an array of the PII keys and instead of putting the values directly in here, do lookups to pull out the headers and columns based on them?